kolibri/plugins/learn/assets/src/my_downloads/views/MyDownloads.vue

```diff
@@ -178,8 +143,12 @@
       removeResources(resources) {
         if (resources.length === 1) {
           this.removeDownloadRequest(resources[0]);
+          Vue.delete(this.downloadRequestMap.value, resources[0].id);
```

Comment:
I do think it would be better to keep this removal in the composable `removeDownloadRequest` function.

Also, `downloadRequestMap` is a reactive, so its value is referenced directly, not via a `.value` property.

Reply:
I think I must be doing something wrong with the reactivity/setup here then, because when I keep it in the composable, it just doesn't work. Not only in the "the page doesn't rerender as expected" sense, but `Vue.delete()` seems to not do anything. logging the value before and after it is the same - nothing is deleted. 

I can see a couple of places where we are using in in mutations, but I couldn't find another place where we are using it in a composable to reference.